### PR TITLE
Update contacts in API references

### PR DESCRIPTION
**Pull request by shraddhavp**
The PR updates the contacts section int he API refrences.

### CODE REVIEW

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

File: docusaurus/docs/api/contact.md [hunk 1]
```diff
@@ -192,41 +187,41 @@ const isOfficial = contact.type() === bot.Contact.Type.Official
...
 
 ```javascript
 const province = contact.province()
 ```
 
-### contact.city\(\) ⇒ `string` \| `null`
+### Contact.city\(\) ⇒ `string` \| `null`
 
-Get the region 'city' from a contact
+The method gets the region 'city' from a contact.Check the below example for implementation.
```
Comment (Soumi7) [inline docusaurus/docs/api/contact.md]: The -> this?

File: docusaurus/docs/api/contact.md [hunk 2]
```diff
@@ -192,41 +187,41 @@ const isOfficial = contact.type() === bot.Contact.Type.Official
...
 
 ```javascript
 const gender = contact.gender() === bot.Contact.Gender.Male
 ```
 
-### contact.province\(\) ⇒ `string` \| `null`
+### Contact.province\(\) ⇒ `string` \| `null`
 
-Get the region 'province' from a contact
+the method gets the region 'province' from a contact.Check the below example for implementation.
```
Comment (Soumi7) [inline docusaurus/docs/api/contact.md]: The -> this

File: docusaurus/docs/api/contact.md [hunk 3]
```diff
@@ -280,17 +273,13 @@ const contactFindByAlias = await bot.Contact.find({ alias:"lijiarui"} )
 
 ### Contact.findAll\(\[queryArg\]\) ⇒ `Promise <Contact []>`
 
-Find contact by `name` or `alias`
-
-If use Contact.findAll\(\) get the contact list of the bot. Include the contacts from bot's rooms.
+The method finds contact by `name` or `alias`.If  you use Contact.findAll\(\) get the contact list of the bot. Include the contacts from bot's rooms.
```
Comment (Soumi7) [inline docusaurus/docs/api/contact.md]: The -> this

**Review by Soumi7 — CHANGES_REQUESTED** [review]

File: docusaurus/docs/api/contact.md (hunk 3 above)
Comment (shraddhavp) [reply docusaurus/docs/api/contact.md]: done

**Comment by shraddhavp** [comment]
@huan

**Review by Soumi7 — APPROVED** [review]
LGTM

**Review by proudofsimin — APPROVED** [review]

**Pull request merged**